nilearn/datasets/atlas.py

```diff
@@ -108,8 +108,8 @@ def fetch_atlas_difumo(
         )
     if resolution_mm not in valid_resolution_mm:
         raise ValueError(
-            "Requested resolution_mm={resolution_mm} is not available. "
-            "Valid options: {valid_resolution_mm}"
+            f"Requested resolution_mm={resolution_mm} is not available. "
```

Comment:
I find it 'worrying' that this 'invalid' f strings (meaning that this should have been an f string and it was just a sting) was not caught by our tests.

is this a ruff autofix thing?

Reply:
These might be **preview** rules, not yet available in the default set of rules. Let me have a look.